src/python/ensembl/io/genomio/seq_region/prepare.py

```diff
@@ -435,9 +435,9 @@ def report_to_csv(report_path: PathLike) -> Tuple[str, Dict]:
 
 
 def prepare_seq_region_metadata(
+    dst_file: PathLike,
```

Comment:
I would still prefer to have this after the input files, to follow the usual `src`, `dst` order, but this is just a preference.

Reply:
@MatBarba forgot to move this one as well with your [last commit](https://github.com/Ensembl/ensembl-genomio/pull/232/commits/20c95f363ee5cb2ec0f61e1d6548ccfb3d15b7a4).